### DISCUSSION TITLE
Support for Japanese retro computers?

### DISCUSSION THREAD

**Discussion by RaouleMenard** [86Box/86Box#3107] [Ideas]
Hello,

I was wondering if support for Japanese retro computers, such as the NEC PC-98 series is in the plan?

They're x86 based with a DOS-ish OS. Also, system ROMs have been dumped.

Thanks!

**Comment by jriwanek** [comment]
It is indeed on our todo list.

**Reply by RaouleMenard** [reply]
> It is indeed on our todo list.

That's great! Looking forward to it!
